### PR TITLE
add warning for incomplete forms

### PR DESCRIPTION
**Pull request by jmtroth0**
@orangejenny (since you know at least a bit about cloudcare) buddy: @Rohit25negi 

https://manage.dimagi.com/default.asp?285003

screen shot 2018-10-29 at 1 15 27 pm

### CODE REVIEW

File: corehq/apps/cloudcare/templates/formplayer/session_list.html [hunk 1]
```diff
@@ -16,6 +16,12 @@
 </script>
 
 <script type="text/template" id="session-view-list-web-apps-template">
+    <div class="module-banner" style="background-color:#ffe3c2;">
```
Comment (orangejenny) [inline corehq/apps/cloudcare/templates/formplayer/session_list.html]: Inline style and hard-coded color values are bad news. How about using `.alert` and `.alert-info`?

File: corehq/apps/cloudcare/templates/formplayer/session_list.html [hunk 2]
```diff
@@ -16,6 +16,12 @@
...
+        <span>
+            Web Apps deletes any Incomplete Forms that are more than 7 days old.
```
Comment (orangejenny) [inline corehq/apps/cloudcare/templates/formplayer/session_list.html]: translate?

**Comment by jmtroth0** [comment]
@orangejenny updated

probably shouldn't have just copy-pasted from [here](https://github.com/dimagi/commcare-hq/commit/aba062a9eb739313dccfdb93eee29d96d3cfe59d#diff-80ca6b55903fc299235ce821204dfc55)

**Review by orangejenny — APPROVED** [review]

File: corehq/apps/cloudcare/templates/formplayer/session_list.html [hunk 3]
```diff
@@ -16,6 +16,14 @@
 </script>
 
 <script type="text/template" id="session-view-list-web-apps-template">
+    <div class="module-banner alert alert-info">
+        <span>
+            {% blocktrans %}
+                Web Apps deletes any Incomplete Forms that are more than 7 days old.
```
Comment (Rohit25negi) [inline corehq/apps/cloudcare/templates/formplayer/session_list.html]: should not this come with trans tag

**Review by Rohit25negi — APPROVED** [review]
left one nit.

File: corehq/apps/cloudcare/templates/formplayer/session_list.html (hunk 3 above)
Comment (jmtroth0) [reply corehq/apps/cloudcare/templates/formplayer/session_list.html]: blocktrans should be doing the same thing, as I understand it

Comment (Rohit25negi) [reply corehq/apps/cloudcare/templates/formplayer/session_list.html]: Yaa correct. overlooked that. All good

**Pull request merged**